src/gravel/controllers/tests/test_config.py

```diff
@@ -13,7 +13,7 @@ def setUp(self):
 
     def test_config_version(self):
         config = Config()
-        assert config.config.version == 2
+        assert config.config.version == 3
```

Comment:
nit, but wondering at what point we should add various samples of config versions to the test ...

Reply:
To be honest, this could have stayed the same all this time. This will only matter, I think, once we start trying to ensure backwards compatibility. I'd say once we cut the first release.